lib/xenops_utils.ml

```diff
@@ -27,6 +27,23 @@ module Unix = struct
 
   let file_descr_of_rpc x = x |> Rpc.int_of_rpc |> file_descr_of_int
   let rpc_of_file_descr x = x |> int_of_file_descr |> Rpc.rpc_of_int
+
+  (** [rm path] removes a file or directory recursively without following
+  * symbolic links. It may raise [Failure] *)
+  let rm path =
```

Comment:
Could you add something to the name to make it clear its recursive? There is also Socket.Unix.rm below which is different from this function I think.

Reply:
i agree. What about `rmtree` like in python?

Reply:
Good suggestion.